### PR TITLE
Add instruction for OpenSearch training

### PR DESCRIPTION
**Pull request by anelook**
To make it easier for training instructors, and if we want to share examples with the attendees, I put them all together

### CODE REVIEW

**Comment by runwuf** [comment]
Thanks @anelook! I'm thinking perhaps examples for training / workshop could go under `workshops` or `trainings`, what do you think? maybe need a discussion in a wider audience? what do you think?
I'll continue my review this weekend.

**Comment by anelook** [comment]
@runwuf , yes, sounds good to me! I didn't see existing folder with trainings, so I created a new one. Let me know if you think we should put it elsewhere

**Review by runwuf — CHANGES_REQUESTED** [review]
Hi @anelook minor suggestions added.
Also, I have created a script to automate services and configs creation.

**Comment by runwuf** [comment]
@anelook - thoughts on changing `open-search-1` directory name to a something like `opensearch-kafka-connect-measurements`? as this example could be re-used in your other workshop / event not specific as `course-1`?

File: trainings/open-search-part-1/buildings.sh
[the remark this answers is not in the document]
Comment (anelook) [reply trainings/open-search-part-1/buildings.sh:1]: Agree, moved it there

File: trainings/open-search-part-1/README.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+To teardown these services and cleanup
+```
+./lab.sh down
+```
+
+### Search queries
+
+For the building examples, find queries to setup mapping, data and run search requests in buildings.sh.
+You can copy the content and past it in OpenSearch Dashboards Dev Tools and then select chunks one by one to run them.
+
```
[the remark this answers is not in the document]
Comment (anelook) [reply trainings/open-search-part-1/README.md:149]: done!

**Review by runwuf — APPROVED** [review]

**Pull request merged**